modules/snooker/src/main/ts/ephox/snooker/api/CellMutations.ts

```diff
@@ -2,13 +2,13 @@ import { SugarElement } from '@ephox/sugar';
 
 import * as Sizes from '../resize/Sizes';
 import * as CellUtils from '../util/CellUtils';
+import { CellElement } from '../util/TableTypes';
 
-const halve = (main: SugarElement, other: SugarElement): void => {
-  const colspan = CellUtils.getSpan(main, 'colspan');
+const halve = (main: SugarElement<CellElement>, other: SugarElement<CellElement>): void => {
   // Only set width on the new cell if we have a colspan of 1 (or no colspan) as we can only safely do that for cells
   // that are a single column, since we don't know the individual column widths for a cell with a colspan.
   // Instead, we'll rely on the adjustments/postAction logic to set the widths based on other cells in the column
-  if (colspan === 1) {
+  if (!CellUtils.hasColspan(main)) {
```

Comment:
Is there a reason this was changed as part of a types PR?

Reply:
I could have sworn I left a comment about this... anyways the problem was this can be passed a `col` or `cell` and the old logic only worked with a cell (technically it would have worked with a col, but would have always returned `1`). So this instead is identical functionality wise, except it properly supports both `col` and `cell`.